traffic_ops/traffic_ops_golang/crconfig/handler.go

```diff
@@ -286,7 +284,7 @@ func SnapshotOldGUIHandler(w http.ResponseWriter, r *http.Request) {
 	}
 
 	if err := Snapshot(inf.Tx.Tx, crConfig, tm); err != nil {
-		writePerlHTMLErr(w, r, inf.Tx.Tx, errors.New(r.RemoteAddr+" making CRConfig: "+err.Error()), err)
+		api.HandleErr(w, r, inf.Tx.Tx, http.StatusInternalServerError, err, errors.New(r.RemoteAddr+" making CRConfig: "+err.Error()))
```

Comment:
The error returned from `crconfig.Snapshot()` is 500-level error and should not be displayed to the user.

Reply:
Fixed now.